examples/reference/elements/bokeh/Labels.ipynb

```diff
@@ -0,0 +1,102 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### **Title**: Labels Element\n",
+    "\n",
+    "**Dependencies**: Bokeh\n",
+    "\n",
+    "**Backends**: [Bokeh](./Labels.ipynb), [Matplotlib]('../matplotlib/Labels.ipynb')"
```

Comment:
Please use the table that is consistent with all the other element notebooks...

Reply:
That does not format nicely in JupyterLab, all notebooks should be converted to this format. It looks nicer too without all the horrible HTML.

Reply:
In that case it should be fixed in a separate PR. Not mixing different markup.

Reply:
I've used the new format in all the recently added elements, no point reverting those. Happy to follow up this PR by updating all existing reference notebooks (although that should probably wait until your metadata PR is merged).

Reply:
Ok. Sounds good.